src/components/MyContactCard.js

```diff
@@ -1,23 +1,14 @@
-import useStore from './Store';
-
-export default function SingelContactCard() {
-	const myContact = useStore(state => state.myContact);
+export default function MyContactCard(contact) {
```

Comment:
Usually we call this object "props", but you can call it "contact", although it is uncommon. 

I would suggest destructuring the object on the spot like this:

`export default function MyContactCard({firstName, lastName, job, company, email, phone, website }) {`

Just to prevent confusion about the naming, but I leave it up to you to decide.